### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by EdTheLoon**
This mainly focuses on cleaning up a lot of the HTML code.
I removed redundant use of the font tags and used CSS to hold this information instead.
I made a slight amendment to the layout of the user controls to the right hand side of the map view.
Re-formatted HTML to use indentations of 2 spaces and used VS Code to auto format.
Fixed a bug in which the pop up form wouldn't show if the checkbox was already checked when the page loaded. This would happen because the page would remember your last setting.

In short: the HTML is easier to read.

### CODE REVIEW

**Review by Sycosys — APPROVED** [review]

**Comment by Sycosys** [comment]
issues seen on android.
